src/components/projects/A32NX/ExtendedFeatures.tsx

```diff
@@ -1,89 +1,126 @@
                 <FeatureBody>
-                    Want to experiment with our new features? You can! We have several ready to test branches with experimental features on our installer!
+                    Want to experiment with our new features? You can! We have several ready to test branches such as Stable, Development and experimental.
+                    The experimental version of the A32NX includes full rewrites of the default autopilot, Fly-by-wire system and much more.
                 </FeatureBody>
             </Feature>
 
-            <Feature key="Constant Additions">
+            <Feature key="Open Source">
+                <IconBrandGithub size={40} stroke={1.5} color="teal" />
                 <FeatureTitle>
-                    Constant Additions
+                    Open Source
                 </FeatureTitle>
                 <FeatureBody>
-                    The A32NX development branches are updated almost daily with brand new features you can test right away!
+                    All the FlyByWire projects such as A32NX is completely open sourced which means anyone can contribute to the project, and it can be freely reused under our licence.
                 </FeatureBody>
             </Feature>
 
-            <Feature key="Documentation">
+            <Feature key="Product Installer">
+                <IconPackage size={40} stroke={1.5} color="teal" />
                 <FeatureTitle>
-                    Documentation
+                    Product Installer
                 </FeatureTitle>
                 <FeatureBody>
-                    FlyByWire has created several tutorials and documents to make learning the A320 quickly and accurately possible for anyone!
+                    FlyByWire has a custom product installer which makes the task of installing all FlyByWire products hassle free.
                 </FeatureBody>
             </Feature>
 
-            <Feature key="Open Source">
+            <Feature key="Simbrief Integration">
+                <IconMap size={40} stroke={1.5} color="teal" />
                 <FeatureTitle>
-                    Open Source
+                    Simbrief Integration
                 </FeatureTitle>
                 <FeatureBody>
-                    The FlyByWire A32NX is completely open sourced, anyone can contribute to the project, and it can be freely reused under our licence.
+                    Make flight plan entry a breeze with the SimBrief integration. All it takes is one click and you are ready to go!
+                </FeatureBody>
+            </Feature>
+
+            <Feature key="Verified by Real Pilots">
+                <IconUserCheck size={40} stroke={1.5} color="teal" />
+                <FeatureTitle>
+                    Verified by Real Pilots
+                </FeatureTitle>
+                <FeatureBody>
+                    The A32NX has been analyzed and tested by hundreds of real A320 pilots for accuracy to the real life aircraft.
```

Comment:
```suggestion
                    The A32NX has been analysed and tested by a wide range of real A320 pilots for accuracy to the real aircraft.
```
Spelling, use of statement "hundreds" is a bit much. Suggested changes